jaxws-ri/runtime/rt/src/main/java/com/sun/xml/ws/fault/SOAPFaultBuilder.java

```diff
@@ -558,4 +564,22 @@ private static JAXBContext createJAXBContext() {
             throw new Error(e);
         }
     }
+
+   private static boolean getBooleanSystemProperty() {
+        final String propertyString = System.getProperty("com.sun.xml.ws.fault.SOAPFaultBuilder.captureExceptionMessage");
+        if (propertyString == null) {
+            //default value 
+            return true;
+        } else {
+            return Boolean.getBoolean(propertyString);
+        }
+    }
+
+   public static boolean getCaptureExceptionMessage() {
+        return captureExceptionMessage;
+   }
+
+   public static void setCaptureExceptionMessage() {
```

Comment:
setter needs an argument, so one can call it and change the value from a test (for example); take a look at the `dump` and similar fields in [HttpAdapter](https://github.com/eclipse-ee4j/metro-jax-ws/blob/master/jaxws-ri/runtime/rt/src/main/java/com/sun/xml/ws/transport/http/HttpAdapter.java#L1086-L1135), the way they are being initialized and set and follow that pattern. Also note that if the method is to return boolean value, then its name should not start with `get` but rather with `is`, `should` or something implying yes/no answer